Gemfile.lock

```diff
@@ -341,7 +341,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.2.2)
       rake
-    rake (12.0.0)
+    rake (11.3.0)
```

Comment:
ok, but this one breaks my installation on FreeBSD (I have rake installed globally there)

````
$ pkg which /usr/local/bin/rake
/usr/local/bin/rake was installed by package rubygem-rake-12.0.0_1
````

Now a simple `bundle install` ends with

````
Bundler::GemspecError: Could not read gem at
/usr/local/lib/ruby/gems/2.3/cache/rake-11.3.0.gem. It may be corrupted.
An error occurred while installing rake (11.3.0), and Bundler cannot
continue.
Make sure that `gem install rake -v '11.3.0'` succeeds before bundling.

In Gemfile:
  rails-settings-cached was resolved to 0.6.5, which depends on
    rails was resolved to 5.0.3, which depends on
      railties was resolved to 5.0.3, which depends on
        rake
````